chronicles/log_output.nim

```diff
@@ -422,40 +415,87 @@ macro append*(o: var AnyOutput,
   result.add newCall("append", o, arg2)
   for arg in restArgs: result.add newCall("append", o, arg)
 
-proc rfcTimestamp: string =
-  now().format("yyyy-MM-dd HH:mm:ss'.'fffzzz")
+import
+  stew/objects
+
+let timezone = toArray(6, now().format("zzz"))
```

Comment:
this needs to be updated regularly (when (time mod 15 minutes) changes afair, some countries are off by 15/45  minutes, but I don't remember if there are other odditites) or you'll miss DST which change the UTC offsets as well as regulatory changes which change the timezone definitions